art/utils/ensemble.py

```diff
@@ -0,0 +1,36 @@
+from art.core import ArtModule
+from art.utils.enums import BATCH, PREDICTION
+
+import torch
+from torch import nn
+
+from typing import List
+from copy import deepcopy
+
+
+class ArtEnsemble(ArtModule):
+    """
+    Base class for ensembles.
+    """
+
+    def __init__(self, models: List[ArtModule]):
+        super().__init__()
```

Comment:
Please add an option to provide model_class + list of checkpoints. But leave the list of models too. Make everything Optional.